code_samples/back_office/online_editor/config/packages/custom_buttons.yaml

```
@@ -0,0 +1,16 @@
+ezplatform:
+    system:
+        admin_group:
+            fieldtypes:
+                ezrichtext:
+                    toolbars:
+                        heading:
+                            buttons:
+                                ezanchor:
+                                    priority: 100
+                                ezembedinline:
+                                    visible: false
+                        ezadd:
+                            buttons:
+                                hr:
+                                    priority: 50
```

```suggestion
                    toolbar:
                        group1:
                            priority: 60
                            buttons:
                                ibexaMoveUp:
                                    priority: 30
                                ibexaMoveDown:
                                    priority: 20
                                    visible: false
                                heading:
                                    priority: 10
```

those changes needs to be added on 4.x